### ISSUE TITLE
No Windows-based integration tests for NB.GV

### ISSUE THREAD

**Issue by qmfrederik** [dotnet/Nerdbank.GitVersioning#526]
The integration tests in CI cover macOS and various Linux distributions, but none appear to cover Windows (and hence net461)

**Comment by AArnott** [comment]
I consider that the unit tests that run on Windows cover it, since some even invoke msbuild on stock projects. The linux and mac distro integration tests are there to make sure it works (at a smoke test level) on xplat.

**Comment by qmfrederik** [comment]
Fair point, though the NuGet package layout is not being tested (i.e. if some assemblies are missing in the `MSBuildFull` directory in the NuGet package, those tests won't catch that).
